packages/devtools_app/lib/src/charts/chart.dart

```diff
@@ -116,6 +123,12 @@ class ChartPainter extends CustomPainter {
 
   @override
   void paint(Canvas canvas, Size size) {
+    // TODO(terry): Used to monitor total painting time. For large
```

Comment:
remove this logging or hide it behind a boolean top level field (eg `bool debugTrackPaintTime=true`)

Reply:
done.